deny.toml

```diff
@@ -0,0 +1,71 @@
+targets = [
+    { triple = "x86_64-unknown-linux-gnu" },
+    { triple = "aarch64-unknown-linux-gnu" },
+    { triple = "armv7-unknown-linux-gnu" },
+]
+
+[advisories]
+db-path = "~/.cargo/advisory-db"
+db-urls = ["https://github.com/rustsec/advisory-db"]
+vulnerability = "deny"
+unmaintained = "warn"
+yanked = "deny"
+notice = "warn"
+ignore = []
+
+[licenses]
+unlicensed = "deny"
+allow = [
+    "Apache-2.0",
+    "BSD-3-Clause",
+    "ISC",
+    "MIT",
+]
+deny = []
+copyleft = "deny"
+allow-osi-fsf-free = "neither"
+default = "deny"
+confidence-threshold = 0.8
+exceptions = [
+    { allow = ["Zlib"], name = "adler32", version = "*" },
+    { allow = ["ISC", "MIT", "OpenSSL"], name = "ring", version = "*" },
+]
+
+[[licenses.clarify]]
+name = "ring"
+version = "*"
+expression = "MIT AND ISC AND OpenSSL"
+license-files = [
+    { path = "LICENSE", hash = 0xbd0eed23 }
+]
+
+[bans]
+multiple-versions = "deny"
+# Wildcard dependencies are used for all workspace-local crates.
+wildcards = "allow"
+highlight = "all"
+deny = []
+skip = [
+    # Pulled in by tracing-futures 0.2
+    { name = "pin-project", version = "0.4.27" },
+    { name = "pin-project-internal", version = "0.4.27" },
+]
+skip-tree = [
+    # `prost-build` uses a version of tempfile that pulls in older versions of
+    # `rand`, etc; and these are only used at build-time, anyway.
+    { name = "tempfile", version = "=3.1.0", depth = 4 },
+]
+
+[sources]
+unknown-registry = "deny"
+unknown-git = "deny"
+allow-registry = ["https://github.com/rust-lang/crates.io-index"]
+allow-git = [
+    "https://github.com/hawkw/tokio-trace",
+    "https://github.com/hyperium/tonic",
+]
+
+[sources.allow-org]
```

Comment:
TIOLI: maybe we should just allow the "tokio", "tower", and "hyperium" orgs globally, to make it less churn  to patch stuff in the future?

Reply:
I'd rather add explicit allowances as we need them. Since we control the linkerd project, though, I'm okay with a blanket allowance for linkerd.